src/libraries/System.DirectoryServices.Protocols/ref/System.DirectoryServices.Protocols.cs

```diff
@@ -382,6 +382,12 @@ public partial class LdapSessionOptions
         internal LdapSessionOptions() { }
         public bool AutoReconnect { get { throw null; } set { } }
         public string DomainName { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("android")]
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("ios")]
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("tvos")]
```

Comment:
These are redundant, the assembly has `<UnsupportedOSPlatforms>browser;android;ios;tvos</UnsupportedOSPlatforms>` attribute
https://github.com/dotnet/runtime/blob/f552567173b706812d11579f8059a14a748dab81/src/libraries/System.DirectoryServices.Protocols/Directory.Build.props#L6C1-L6C78
```suggestion
```

Reply:
Removed from source but not from ref

Reply:
I'm still seeing these, @steveharter.  I think we can omit the attributes already defined at the assembly level, and just add the one additional `windows` one here?

Reply:
Nevermind, for some reason was looking at subset of commits.

Reply:
I'm assuming the previously excluded `browser;android;ios;tvos` were because the browser and mobile platforms don't support LDAP \ DirectoryServices.